Detectors/ZDC/calib/src/BaselineCalibSpec.cxx

```diff
@@ -115,6 +128,40 @@ void BaselineCalibSpec::sendOutput(o2::framework::DataAllocator& output)
   output.snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "ZDCBaselinecalib", 0}, info);         // root-serialized
   // TODO: reset the outputs once they are already sent (is it necessary?)
   // mWorker.init();
+
+  o2::zdc::CalibParamZDC& opt = const_cast<o2::zdc::CalibParamZDC&>(CalibParamZDC::Instance());
```

Comment:
The ConfigKeyValue object must be const ref, why do you constast it?
Please use  `const auto&  opt = CalibParamZDC::Instance();`, also in other places.